Tetrahedral_remeshing/examples/Tetrahedral_remeshing/mesh_and_remesh_with_adaptive_sizing.cpp

```diff
@@ -0,0 +1,101 @@
+#define CGAL_TETRAHEDRAL_REMESHING_VERBOSE
+
+#include <CGAL/Exact_predicates_inexact_constructions_kernel.h>
+
+#include <CGAL/Polyhedron_3.h>
+#include <CGAL/Polyhedral_mesh_domain_3.h>
+
+#include <CGAL/Mesh_triangulation_3.h>
+#include <CGAL/Mesh_complex_3_in_triangulation_3.h>
+#include <CGAL/Mesh_criteria_3.h>
+#include <CGAL/make_mesh_3.h>
+
+#include <CGAL/tetrahedral_remeshing.h>
+#include <CGAL/Tetrahedral_remeshing/Adaptive_remeshing_sizing_field.h>
+
+#include <CGAL/IO/File_medit.h>
+
+// Domain
+using K = CGAL::Exact_predicates_inexact_constructions_kernel;
+using Point = K::Point_3;
+using FT = K::FT;
+using Polyhedron = CGAL::Polyhedron_3<K>;
+using Mesh_domain = CGAL::Polyhedral_mesh_domain_3<Polyhedron, K>;
+
+// Triangulation for Meshing
+using Tr = CGAL::Mesh_triangulation_3<Mesh_domain>::type;
+using C3t3 = CGAL::Mesh_complex_3_in_triangulation_3<Tr>;
+
+// Criteria
+using Mesh_criteria = CGAL::Mesh_criteria_3<Tr>;
+
+// Triangulation for Remeshing
+using T3 = CGAL::Triangulation_3<Tr::Geom_traits,
+                                 Tr::Triangulation_data_structure>;
+
+// To avoid verbose function and named parameters call
+using namespace CGAL::parameters;
+
+int main(int argc, char* argv[])
+{
+  const std::string fname = (argc > 1) ? argv[1] : CGAL::data_file_path("meshes/elk.off");
```

Comment:
While I removed `#include <string>`  in an example where it was not used, I did not add it here. Shall I ?  @sloriot

Reply:
probably safer as you then won't depend on what is done in other headers.